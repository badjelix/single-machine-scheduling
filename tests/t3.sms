30
0 2 2 1 2
9 2 11 2 1 1
9 3 15 1 3
6 1 7 1 1
15 12 51 3 3 4 5
2 4 14 3 1 2 1
6 2 10 2 1 1
17 7 38 5 1 1 2 2 1
1 6 7 4 1 1 2 2
7 12 19 3 4 3 5
18 10 38 3 6 1 3
13 4 25 3 2 1 1
11 9 38 5 1 2 1 2 3
11 1 14 1 1
6 8 22 2 5 3
7 7 28 3 3 2 2
3 6 21 2 2 4
18 10 28 3 5 2 3
9 3 12 2 1 2
0 9 27 5 1 2 2 2 2
2 15 17 4 4 4 3 4
13 15 43 5 3 2 2 3 5
7 4 11 3 1 2 1
2 2 6 2 1 1
13 14 55 3 4 5 5
2 4 6 2 2 2
15 9 42 2 6 3
10 7 31 4 1 1 3 2
19 8 43 5 3 1 1 1 2
7 2 9 2 1 1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
