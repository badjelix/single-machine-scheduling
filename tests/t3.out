12
1 0
2 9 10
3 12
4 6
7 7 8
8 20 25 32 34 37
14 11
17 4 15
24 2 3
25 41 45 50
28 19 24 26 29
29 21 31 36 38 39
